test/data/keymaps/real-modifiers.xkb

```diff
@@ -0,0 +1,164 @@
+xkb_keymap {
+xkb_keycodes "test" {
+    minimum = 8;
+    maximum = 255;
+	<LVL3>  = 92;
+    <LFSH>  = 50;
+	<RTSH>  = 62;
+	<LALT>  = 64;
+	<RALT>  = 108;
+	<LWIN>  = 133;
+	<RWIN>  = 134;
+	<LCTL>  = 37;
+	<RCTL>  = 105;
+	<CAPS>  = 66;
+
+	<AD01> = 24;
+	<AD02> = 25;
+	<AD03> = 26;
+	<AD04> = 27;
+	<AD05> = 28;
+	<AD06> = 29;
+	<AD07> = 30;
+	<AD08> = 31;
+	<AD09> = 32;
+};
+
+xkb_types "complete" {
+    // Define a buch of virtual modifier to test mappings to Mod5-Mod10 real
+    // modifiers. WonderMod will be mapped to Mod5+Mod6.
+    virtual_modifiers MyMod5,MyMod6,MyMod6,MyMod7,MyMod8,MyMod9,MyMod10,WonderMod;
```

Comment:
Another nitpick because I'm getting confused reading this: how about `VMod5` etc. so it's instantly readable as "this is a virtual modifier?

Reply:
Let’s do this.